src/scss/components/_blockquote.scss

```diff
@@ -0,0 +1,6 @@
+.docs-b-Blockquote {
+  background-color: $sprk-alert-bg-color-info;
```

Comment:
I love this blockquote style with that lighter blue! It's cute! Can you update this to use the blue color itself and not the color from the alert, just in case we change the alert color in Spark. Don't want to then have that accidentally change our Blockquotes in our NDS.

Reply:
yes good catch! I meant to change this but forgot.